lib/docker-sync/sync_strategy/unison-unox.rb

```diff
@@ -0,0 +1,204 @@
+require 'thor/shell'
+require 'docker-sync/preconditions'
+require 'docker-sync/execution'
+require 'open3'
+require 'socket'
+require 'terminal-notifier'
+
+module Docker_Sync
+  module SyncStrategy
+    class Unison_Unox
+      include Thor::Shell
+      include Execution
+
+      @options
+      @sync_name
+      @watch_thread
+      @local_server_pid
+      UNISON_CONTAINER_PORT = '5000'
+      def initialize(sync_name, options)
+        @options = options
+        @sync_name = sync_name
+        # if a custom image is set, apply it
+        if @options.key?('image')
+          @docker_image = @options['image']
+        else
+          @docker_image = 'eugenmayer/unison:unox'
+        end
+        begin
+          Preconditions::unison_available
+          Preconditions::unox_available
```

Comment:
@onnimonni thanks for trying to help, but I have implemented a precondition to check that `unox` is accessible from the `docker-sync` process.